### ISSUE TITLE
[W1 思考题 6] VAE 的问题：解码器方差、模糊、posterior collapse

### ISSUE THREAD

**Issue by qugank** [Qi-StarterTrain/diffusion-models-starter-materials#8]
## 涉及内容

derive_01 §6.4、§7（公式与代码）

## 题目（找茬题）

derive_01 §6.4 的框中公式写道"去掉了无关常数（$\frac{1}{2}$ 等可吸收到 learning rate）"，§7 又给出了 `vae_loss` 的实现。

1. 从 $-\mathrm{ELBO} = \frac{1}{2}\|x-\mu_\theta(z)\|^2 + \frac{1}{2}\sum_i(\mu_i^2+\sigma_i^2-\log\sigma_i^2-1) + \text{const}$ 出发，检查 §6.4 的框中公式和 §7 的 `vae_loss` 代码**是否都与它只差一个整体常数倍**。
2. 如果有不一致，它相当于思考题 5（#7）里哪个 $\beta$？对训练结果会有什么影响？
3. "常数可以吸收进 learning rate"这句话，什么时候成立、什么时候不成立？
4. 实践中 `F.mse_loss` 用 `reduction='sum'` 还是 `'mean'` 会不会改变 recon 与 KL 的相对权重？

---

**作答方式**：直接在本 Issue 下评论作答（公式用 `$...$` / `$$...$$`，手写拍照也可以）。先独立推一遍再看别人的回答；看到别人的回答有漏洞或有更简洁的写法，欢迎直接回复讨论。

> 配套资源：⭐ [VAE 讲解视频（YouTube）](https://www.youtube.com/watch?v=NlIqjtbjjRE)——讲 VAE 讲得最系统，如果只看一个就看这个。
